### PR TITLE
Implement TryFromChunks for chrono FixedOffset and UTC

### PR DESCRIPTION
**Pull request by reneklacan**
https://github.com/murar8/axum_typed_multipart/issues/47

I couldn't figure out a way to make it fully general to accept any TimeZone

### CODE REVIEW

File: src/try_from_chunks.rs [hunk 1]
```diff
@@ -159,6 +160,50 @@ impl TryFromChunks for Uuid {
...
+impl TryFromChunks for DateTime<Utc> {
+    async fn try_from_chunks(
+        chunks: impl Stream<Item = Result<Bytes, TypedMultipartError>> + Send + Sync + Unpin,
+        metadata: FieldMetadata,
+    ) -> Result<Self, TypedMultipartError> {
+        let field_name = get_field_name(&metadata.name);
+        let bytes = Bytes::try_from_chunks(chunks, metadata).await?;
+        let body_str =
+            std::str::from_utf8(&*bytes).map_err(|err| TypedMultipartError::WrongFieldType {
+                field_name: field_name.clone(),
```
Comment (murar8) [inline src/try_from_chunks.rs:195]: I guess this could be a `mem::take`?

File: src/lib.rs [hunk 2]
```diff
@@ -31,6 +31,8 @@
 //! - [char]
 //! - [String]
 //! - [axum::body::Bytes]
+//! - [chrono::DateTime<chrono::FixedOffset>]
```
Comment (murar8) [inline src/lib.rs:34]: I would add the version here just in case.

**Review by murar8 — COMMENTED** [review]
Hi, thank you very much for your contribution! I left a couple minor comments but overall looks good!

**Comment by murar8** [comment]
Don't mind the failing CI, will have it fixed tonight hopefully.

File: src/lib.rs (hunk 2 above)
Comment (reneklacan) [reply src/lib.rs:34]: you mean the version of chrono?

**Review by murar8 — APPROVED** [review]

**Pull request merged**